mafia/data/garbo_item_lists.json

```diff
@@ -230,5 +230,53 @@
     "Angry Farmer's Wife Candy",
     "cotton candy pillow",
     "sugar shillelagh"
+  ],
+  "trainset": [
```

Comment:
are we -positive- these are the only candies from the trainset? i can't recall getting anything else but i don't remember where we sourced this stuff

Reply:
I got this list from Nas, he posted it December 29 from having run it all season. This isn't set in stone, though, we can edit her when relevant